4788/src/main/include/RobotMap.h

```diff
@@ -115,8 +115,17 @@ struct RobotMap {
     wml::control::PIDGains gainsVelocity{"Drivetrain Velocity", 1};
     wml::Drivetrain drivetrain{drivetrainConfig, gainsVelocity};
 
+
+    
+
   }; DrivebaseSystem drivebaseSystem;
 
+  struct ClimberSystem {
+    wml::actuators::DoubleSolenoid leftClimberSolenoid{ ControlMap::PCModule, wml::actuators::PneumaticsModuleType::kCTRE,ControlMap::leftClimberPort1, ControlMap::leftClimberPort2, 0.1};
+    wml::actuators::DoubleSolenoid rightClimberSolenoid{ ControlMap::PCModule, wml::actuators::PneumaticsModuleType::kCTRE,ControlMap::rightClimberPort1, ControlMap::rightClimberPort2, 0.1};
+  }; ClimberSystem climberSystem;
+};
```

Comment:
Extra bracket (code assumes end of robotmap) causes fail

Reply:
Fixed